src/tools/perf/perftest.c

```diff
@@ -482,6 +493,8 @@ static void usage(const struct perftest_context *ctx, const char *program)
     printf("     -E <mode>      wait mode for tests\n");
     printf("                        poll       : repeatedly call worker_progress\n");
     printf("                        sleep      : go to sleep after posting requests\n");
+    printf("     -H <size>      active message header size (%zu), not included in message size\n",
+                                ctx->params.super.ucp.am_hdr_size);
```

Comment:
-H option should be in "Common" section, and the difference between UCT and UCP should be explained in description. Otherwise, there should be two different options.

Reply:
ok

Reply:
@rakhmets looks like better keep it as it is now - it looks clearer. Also we already have the same opt with different meaning for UCP and UCT: `-D`

Reply:
Ok, I see.